code/modules/antagonists/heretic/status_effects/debuffs.dm

```diff
@@ -256,3 +254,138 @@
+	if(!ishuman(owner))
+		return FALSE
+	if(owner.reagents.has_reagent(/datum/reagent/water/holywater))
+		return FALSE
+	return TRUE
+
+/atom/movable/screen/alert/status_effect/eldritch_painting
+	name = "Rick Roll'd"
+	desc = "Fucking coders are at it again."
+	icon_state = "eldritch_painting_debug"
+
+//"The Sister and He Who Wept": /obj/structure/sign/painting/eldritch
+/datum/status_effect/eldritch_painting/weeping
+	id = "painting_weeping"
+	alert_type = /atom/movable/screen/alert/status_effect/eldritch_painting/weeping
+	tick_interval = 10 SECONDS
+
+/datum/status_effect/eldritch_painting/weeping/tick(seconds_between_ticks)
+	if(owner.stat != CONSCIOUS || owner.IsSleeping() || owner.IsUnconscious())
+		return
+	if(HAS_TRAIT(owner, TRAIT_ELDRITCH_PAINTING_EXAMINE))
+		return
+
+	owner.cause_hallucination(/datum/hallucination/delusion/preset/heretic, "Caused by The Weeping status effect")
+	owner.add_mood_event("eldritch_weeping", /datum/mood_event/eldritch_painting/weeping)
+
+/atom/movable/screen/alert/status_effect/eldritch_painting/weeping
+	name = "The Sister and He Who Wept"
+	desc = "The weeping echos through your mind like an echo, undoing your psyche! Maybe if you look at the painting again, it won't hurt so badly..."
+	icon_state = "eldritch_painting_weeping"
+
+//"The First Desire": /obj/structure/sign/painting/eldritch/desire
+/datum/status_effect/eldritch_painting/desire
+	id = "painting_desire"
+	alert_type = /atom/movable/screen/alert/status_effect/eldritch_painting/desire
+	/// How much faster we loose hunger
+	var/hunger_rate = 15
+
+/datum/status_effect/eldritch_painting/desire/on_apply()
+	if(IS_HERETIC_OR_MONSTER(owner))
+		return FALSE
+	if(!ishuman(owner))
+		return FALSE
+	if(HAS_TRAIT(owner, TRAIT_NOHUNGER))
+		return FALSE
+
+	// Allows them to eat faster, mainly for flavor
+	ADD_TRAIT(owner, TRAIT_VORACIOUS, TRAIT_STATUS_EFFECT(id))
+	ADD_TRAIT(owner, TRAIT_FLESH_DESIRE, TRAIT_STATUS_EFFECT(id))
+	return TRUE
+
+/datum/status_effect/eldritch_painting/desire/tick(seconds_between_ticks)
+	if(HAS_TRAIT(owner, TRAIT_ELDRITCH_PAINTING_EXAMINE))
+		return
+	// Causes them to need to eat at 10x the normal rate
+	owner.adjust_nutrition(-hunger_rate * HUNGER_FACTOR)
+	if(SPT_PROB(10, seconds_between_ticks))
+		to_chat(owner, span_notice(pick("You can't stop thinking about raw meat...", "You **NEED** to eat someone.", "The hunger pangs are back...", "You hunger for flesh.", "You are starving!")))
+	owner.overeatduration = max(owner.overeatduration - 200 SECONDS, 0)
+
+/datum/status_effect/eldritch_painting/desire/on_remove()
+	REMOVE_TRAIT(owner, TRAIT_VORACIOUS, TRAIT_STATUS_EFFECT(id))
+	REMOVE_TRAIT(owner, TRAIT_FLESH_DESIRE, TRAIT_STATUS_EFFECT(id))
+	return ..()
+
+/atom/movable/screen/alert/status_effect/eldritch_painting/desire
+	name = "The First Desire"
+	desc = "Your are struck with a ravenous hunger! SATIATE IT AT ANY COST! Or maybe just go start at the painting and long for the excellent meal it promises..."
```

Comment:
```suggestion
	desc = "Your are struck with a ravenous hunger! SATIATE IT AT ANY COST! Or maybe just go stare at the painting and long for the excellent meal it promises..."
```

I'm also wondering if we _want_ status alerts for these to be honest, or at least for all of the ones which have them.  
There's something to say for what's going on to be a little less clear than it will be now that an icon pops up on your screen and basically tells you waht to do.